rest_framework_simplejwt/views.py

```diff
@@ -28,10 +37,65 @@ def post(self, request, *args, **kwargs):
         except TokenError as e:
             raise InvalidToken(e.args[0])
 
-        return Response(serializer.validated_data, status=status.HTTP_200_OK)
+        response = Response(serializer.validated_data, status=status.HTTP_200_OK)
+
+        if api_settings.AUTH_COOKIE:
+            csrf.get_token(self.request)
+            response = self.set_auth_cookies(response, serializer.validated_data)
+
+        return response
```

Comment:
Do we want the payload to be both in the body, and the cookie?

Reply:
Yes, the same API may be used by web and non-web clients.

Reply:
Returning only the cookies might be good as an optional setting. Seems like returning the tokens in the response data implies that the API trusts that an SPA will do the right thing. There are lots of examples and tutorials out there that parse the data and store the tokens in localStorage.
P.S., thanks for this work!